Sources/SwiftProtobuf/JSONDecoder.swift

```diff
@@ -713,19 +707,20 @@ internal struct JSONDecoder: Decoder {
     messageType: Message.Type,
     fieldNumber: Int
   ) throws {
-    if let ext = extensions?[messageType, fieldNumber] {
-      var fieldValue = values[fieldNumber]
-      if fieldValue != nil {
-        try fieldValue!.decodeExtensionField(decoder: &self)
-      } else {
-        fieldValue = try ext._protobuf_newField(decoder: &self)
-      }
-      if fieldValue != nil {
-        values[fieldNumber] = fieldValue
-      } else {
-        // This most likely indicates a bug in our extension support.
-        throw TextFormatDecodingError.internalExtensionError
-      }
+    guard let ext = extensions?[messageType, fieldNumber] else {
+      return
```

Comment:
Is this an internal error since the extension wasn't found?

Or I guess we never get here because we never would have been able to look up the field number in the first place?  So maybe `assert`?

Reply:
I changed this to a simple force-unwrap, which is essentially an `assert`